src/components/Grid/Grid.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+
+
+export default function GridForm() {
+
+    return (<body>
+             <div style={{ padding: 20 }}>
+               <Grid container container direction="row" justify="center" alignItems="center">
+                 //Grid
```

Comment:
<Grid item> should be added at least
current implementation leads to misunderstooding - "//Grid" with center alignment is shown on the page

Reply:
fixed, please review